src/promnesia/sources/pocket.py

```diff
@@ -7,22 +7,43 @@
 def index() -> Results:
     from . import hpi
     from my.pocket import articles
+
     # TODO use docstring from my. module? E.g. describing which pocket format is expected
 
     for a in articles():
-        loc = Loc.make(title='pocket', href=a.pocket_link)
+        title = a.json.get('resolved_title', 'pocket')
+        loc = Loc.make(title=title, href=a.pocket_link)
+        # Add a reverse locator so that the Promnesia browser extension shows a
+        # link on the Pocket page back to the original URL.
+        loc_rev = Loc.make(title=title, href=a.url)
         hls = a.highlights
         if len(hls) == 0:
             yield Visit(
                 url=a.url,
                 dt=a.added,
-                context=None,
+                context=a.json.get('excerpt', None),
                 locator=loc,
             )
+            # Add a visit with the Pocket URL so that we can see that we have
```

Comment:
Hmm, I had thought about adding it at first, but then thought it is pointless because when you're on Pocket website, you can see the original URL in the interface. Do you think this adds some value? I don't mind really, just curious what I'm missing because a similar pattern is also applicable to other data sources like Instapaper :)

Reply:
Looking at this, I agree with you - it is pretty useless. I'll remove it.